### ISSUE TITLE
Lots of warnings after installing 1.5

### ISSUE THREAD

**Issue by mm98** [audiconnect/audi_connect_ha#368]
**Describe the bug**
After installing v1.5 i see lots of warnings for points, that don't match my car (looks like mostly e-tron / EV data). I have an Audi A5 Sportback. I just wanted to report these.

```
This error originated from a custom integration.

Logger: custom_components.audiconnect.audi_models
Source: custom_components/audiconnect/audi_models.py:295
integration: Audi Connect (documentation, issues)
First occurred: 06:23:09 (154 occurrences)
Last logged: 08:53:03

Value for 'plugState' is None; not appending state.
Value for 'plugLockState' is None; not appending state.
Value for 'externalPower' is None; not appending state.
Value for 'plugledColor' is None; not appending state.
Value for 'climatisationState' is None; not appending state.
```

```
This error originated from a custom integration.

Logger: custom_components.audiconnect.audi_models
Source: custom_components/audiconnect/audi_models.py:297
integration: Audi Connect (documentation, issues)
First occurred: 06:23:09 (11 occurrences)
Last logged: 08:53:03

Timestamp for 'remainingChargingTime' is None or missing; not appending state.
```

```
This error originated from a custom integration.

Logger: custom_components.audiconnect.audi_models
Source: custom_components/audiconnect/audi_models.py:337
integration: Audi Connect (documentation, issues)
First occurred: 06:23:09 (11 occurrences)
Last logged: 08:53:03

Value for 'ADBLUE_RANGE' is missing; not appending field.
```

**To Reproduce**
None

**Expected behavior**
None

**Screenshots**
None

**Logfile**
[home-assistant_audiconnect_2024-04-12T07-00-33.383Z.log](https://github.com/audiconnect/audi_connect_ha/files/14955078/home-assistant_audiconnect_2024-04-12T07-00-33.383Z.log)

**Comment by Kolbi** [comment]
From which version did you upgrade to 1.5?

**Comment by Kolbi** [comment]
@coreywillwhat I guess it's related to the _LOGGER.warning introduced by https://github.com/audiconnect/audi_connect_ha/pull/356 - maybe we should change to _LOGGER.debug :)

**Comment by coreywillwhat** [comment]
all these rules! lol Lets call it growing pains.

**Comment by coreywillwhat** [comment]
PR submitted #369 , thanks for reporting @mm98

**Comment by Kolbi** [comment]
Merged PR and creating 1.5.2 in a few minutes.

**Comment by mm98** [comment]
> From which version did you upgrade to 1.5?

From v1.4.0. Thanks for the fix!
